static/config/99datadog.config

```diff
@@ -3,7 +3,9 @@ option_settings:
     - namespace:  aws:elasticbeanstalk:application:environment
       option_name:  DD_API_KEY
       value:  "<YOUR_DD_API_KEY>"
-
+    - namespace:  aws:elasticbeanstalk:application:environment
+      option_name:  DD_AGENT_VERSION
+      value:  "" # For example, "6.15.1", leave empty to install the latest version.
```

Comment:
If users set a value that does not correspond to the major version installed by default, they will also need to change the `baseurl` in the [repository file](https://github.com/DataDog/documentation/pull/5966/files#diff-13dacd0da828fde7e9003ec2de5dc109R21-R32). This should be added as a comment here and in the related docs.

Reply:
We are only doing Agent 6 here. I have another PR to add Agent 7 support [1] and my plan was to stop supporting Agent 6 instead of letting users install either version. The rationale is: this file is something you download when setting up DataDog for the first time and then you keep your own copy of it (so existing users shouldn't use what's in here).

[1] https://github.com/DataDog/documentation/pull/5944/files

Reply:
I see; the thing I'm worried about is customers seeing this and trying to put an Agent 5 version here, which will make the deployment fail.

Reply:
We can mention only agent 6 is supported.